composer.json

```diff
@@ -14,16 +14,17 @@
     ],
     "require": {
         "php":                     "^5.6||^7.0",
-        "symfony/form":            "^2.7||^3.0",
-        "symfony/http-foundation": "^2.7||^3.0"
+        "symfony/form":            "^2.7||^3.0||^4.0",
+        "symfony/http-foundation": "^2.7||^3.0||^4.0"
     },
     "require-dev": {
         "hostnet/phpcs-tool":     "^4.1",
         "phpunit/phpunit":        "^5.4",
-        "symfony/debug":          "^2.7||^3.0",
-        "symfony/phpunit-bridge": "^2.7||^3.0"
+        "symfony/debug":          "^2.7||^3.0||^4.0",
+        "symfony/phpunit-bridge": "^2.7||^3.0||^4.0"
     },
-    "minimum-stability": "stable",
+    "minimum-stability": "dev",
+    "prefer-stable": true,
```

Comment:
Whoa this is scary. Why not only enable this when testing against Symfony 4 in travis?

Reply:
The bundle is actually using this as well: https://github.com/hostnet/form-handler-bundle/blob/master/composer.json#L48-L49

What are the possible side effects of this?

Reply:
It only affects the root package. So no real problem here.

Testing matrix could actually be improved, but I don't have time for that 😅
Thanks for the release 👍

Reply:
This is a root-only setting and applies to all packages. So you might become dependent on a non stable version (dev, not even beta!) of a package without knowing that (as your tests all run in allow-dev mode).

Yet, as soon as you require it in an application, users might also need to add `package: @dev` to make it possible to use/install your package.

So better safe then sorry: Use strict whitelisting of dev versions (preferably beta) and only do that in the job that need it (usually, you test lower + stable + dev versions, with dev on allowed failure).

Reply:
@sstok  Being root only is actually the problem. Add (hypothetically) `"symfony/phpunit-bridge": "4.0"` to the require section and you see what I mean (tests pass, but requiring in an app is no longer possible).